src/sentry/snuba/metrics/query_builder.py

```diff
@@ -84,14 +85,36 @@
 from sentry.utils.snuba import parse_snuba_datetime
 
 
-def parse_field(field: str) -> MetricField:
+def parse_field(field: str, allow_mri: bool = False) -> MetricField:
```

Comment:
Why are you not using `allow_mri`? Maybe you missed it

Reply:
Yep, good catch!